overlord/configstate/configcore/export_managers_test.go

```diff
@@ -0,0 +1,59 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
```

Comment:
I wonder if export_(runwith)state_test.go would be a more consistent naming for this one?

Reply:
file moved